src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -276,98 +238,6 @@ internal NodeViewModel NodeViewModel
             get => _node;
         }
         
-        /// <summary>
-        /// Shows or hides the Use Levels and Keep List Structure checkboxes
-        /// in the node chevron popup menu.
-        /// </summary>
-        public bool UseLevelCheckBoxVisibility
-        {
-            get => _port.PortType == PortType.Input;
-        }
-
-        /// <summary>
-        /// Shows or hides the Use Default Value checkbox in the node chevron popup menu.
-        /// </summary>
-        public bool UseDefaultValueCheckBoxVisibility
-        {
-            get => _port.PortType == PortType.Input && DefaultValueEnabled;
-        }
-
-        /// <summary>
-        /// Shows or hides the Break Connections, Hide Wires and UnhideWires buttons in the node chevron popup menu.
-        /// </summary>
-        public bool OutputPortConnectionsButtonsVisible
-        {
-            get => _port.PortType == PortType.Output;
-        }
-
-        /// <summary>
-        /// Enables or disables the Break Connections button on the node output port context menu.
-        /// </summary>
-        public bool OutputPortBreakConnectionsButtonEnabled
-        {
-            get => OutputPortConnectionsButtonsVisible && IsConnected;
-        }
-
-        /// <summary>
-        /// Determines whether the output port button says 'Hide Wires' or 'Show Wires'
-        /// </summary>
-        public string ShowHideWiresButtonContent
-        {
-            get => showHideWiresButtonContent;
-            set
-            {
-                showHideWiresButtonContent = value;
-                RaisePropertyChanged(nameof(ShowHideWiresButtonContent));
-            }
-        }
```

Comment:
And these.

Reply:
Same

Reply:
There's a good chance I added most of the properties being flagged for deletion!
Since half of them relate to whether the port's an input/output it should be ok. However the ShowHideWiresButtonContent I think will still be useful.